tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -341,6 +341,7 @@ private static void GenerateCoverageFile(ApiMetadata api, string directory)
             var targetExecutable = new XElement("TargetExecutable", "/Program Files/dotnet/dotnet.exe");
             var targetArguments = new XElement("TargetArguments",
                 $"test --no-build -f {GetPreferredCoverageFramework(api)} -c Release");
+            var filters = new XElement("Filters", new XElement("IncludeFilters", new XElement("FilterEntry", new XElement("ModuleMask", api.Id))));
```

Comment:
so this will exclude coverage that you might have from a test in a different folder?  I think that will be true indefinitely for spanner integration tests against V1 (its also true for unit tests, but that part is fixable).  Can we only have this filter for unit tests?